base_layer/core/src/covenants/byte_codes.rs

```diff
@@ -50,7 +50,7 @@ pub const ARG_BYTES: u8 = 0x09;
 pub const ARG_OUTPUT_TYPE: u8 = 0x0a;
 
 //---------------------------------- FILTER byte codes --------------------------------------------//
-
+/// Checks if a byte value results in a valid argument byte code
```

Comment:
Maybe add some docs for these?
- `impl Covenant`
- `pub(super) const ALL_ARGS: [u8; 10] = [`
- `pub(super) const ALL_FILTERS: [u8; 10] = [`